src/sentry/region_to_control/consumer.py

```diff
@@ -84,6 +85,15 @@ def _handle_audit_log_event(self, audit_log_entry: AuditLogEvent, reentry=False)
             error_message = str(e)
             if '"sentry_organization"' in error_message:
                 metrics.incr("region_to_control.consumer.audit_log_entry.stale_event")
+                with sentry_sdk.push_scope() as scope:
+                    scope.level = "warning"
+                    scope.set_tag("error_message", error_message)
+                    scope.set_tag("organization_id", audit_log_entry.organization_id)
+                    scope.set_tag("event_id", audit_log_entry.event_id)
+                    scope.set_tag("actor_label", audit_log_entry.actor_label)
```

Comment:
This could have PII of our customers. Do we need this if we have the event_id and organization_id?

Reply:
Good call, I'll remove.